### ISSUE TITLE
Adding validation loss to progress bar when doing a model fit

### ISSUE THREAD

**Issue by tabacof** [Accenture/AmpliGraph#102]
**Background and Context**
Currently only the average training loss is shown, which is usually not human interpretable. Keras, for example, shows the validation loss on its progress bar.

**Description**
Adding the validation loss to the progress bar message, if early stopping is set up.

**Comment by tabacof** [comment]
Solved: https://github.com/Accenture/AmpliGraph/commit/6252b63c06d3463c441b3cef816695134132fa94
